aspnetcore/fundamentals/minimal-apis/webapplication.md

```diff
@@ -0,0 +1,13 @@
+---
+title: WebApplication and WebApplicationBuilder in Minimal API applications
+author: rick-anderson
+description: Learn about how to use WebApplication and WebApplicationBuilder.
+ms.author: riande
+monikerRange: '>= aspnetcore-7.0'
+ms.date: 10/31/2022
+uid: fundamentals/minimal-apis/webapplication
+---
+
+# WebApplication and WebApplicationBuilder
```

Comment:
Ditto the commentary below on the TOC. This title doesn't feel particularly helpful...